examples/tutorial/vertical_federated_learning/cifar10-splitnn/src/splitnn/cifar10_learner_splitnn.py

```diff
@@ -0,0 +1,614 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from timeit import default_timer as timer
+
+import numpy as np
+import torch
+import torch.optim as optim
+from cifar10_splitnn_dataset import CIFAR10SplitNN
+from torch.utils.tensorboard import SummaryWriter
+from torchvision import transforms
+
+from nvflare.apis.dxo import DXO, from_shareable
+from nvflare.apis.fl_constant import FLContextKey, ReturnCode
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable, make_reply
+from nvflare.apis.signal import Signal
+from nvflare.app_common.abstract.learner_spec import Learner
+from nvflare.app_common.app_constant import AppConstants
+from nvflare.app_common.workflows.splitnn_workflow import SplitNNConstants, SplitNNDataKind
+from nvflare.app_opt.pt.decomposers import TensorDecomposer
+from nvflare.fuel.f3.stats_pool import StatsPoolManager
```

Comment:
This PR will need to wait for #1352 for stats_pool

Reply:
#1352 was merged.